reference/api/multi_search.mdx

```diff
@@ -65,6 +65,10 @@ Each search result object is composed of the following fields:
 
 <CodeSamples id="multi_search_1" />
 
+<Capsule intent="warning">
+If one or more sent queries have an error, Meilisearch will not return any results. In case of multiple errors, the error message will only address the first erroneous query.
```

Comment:
```suggestion
If one or more of the multi-search queries cause an error, Meilisearch will not return any results. In case of multiple errors, the error message found in the task object will describe only the first error encountered.
```

WDYT of this change? For me it was unclear that the error message would be found in a task object. I also think that "erroneous query" implies that the query was badly formatted, whereas an error could conceivably come from something else.

Reply:
That sounds good to me Maryam!

Reply:
I removed the task object part as Meilisearch only shows the error object:
```
{"message":"Unknown field `inexUid` inside `.queries[2]`: did you mean `indexUid`? expected one of `indexUid`, `q`, `offset`, `limit`, `page`, `hitsPerPage`, `attributesToRetrieve`, `attributesToCrop`, `cropLength`, `attributesToHighlight`, `showMatchesPosition`, `filter`, `sort`, `facets`, `highlightPreTag`, `highlightPostTag`, `cropMarker`, `matchingStrategy`",
"code":"bad_request",
"type":"invalid_request",
"link":"https://docs.meilisearch.com/errors#bad_request"}